src/main/java/redis/clients/jedis/JedisFactory.java

```diff
@@ -19,17 +19,13 @@
  * PoolableObjectFactory custom impl.
  */
 class JedisFactory implements PooledObjectFactory<Jedis> {
-  private final AtomicReference<HostAndPort> hostAndPort = new AtomicReference<HostAndPort>();
-  private final int connectionTimeout;
-  private final int soTimeout;
+  
+  private final JedisSocketFactory jedisSocketFactory;
```

Comment:
Shouldn't we use `AtomicReference`, may be in JedisSocketFactory, to match previous implementation?

Reply:
@sazzad16 I'm not sure why it was there to begin with

Reply:
@gkorland #681